config/config.exs

```diff
@@ -33,7 +33,7 @@ config :phoenix, :json_library, Jason
 
 config :school_house, SchoolHouseWeb.Gettext,
   default_locale: "en",
-  locales: ~w(ar bg bn de en es fr gr id it ja ko ms no pl pt ru sk ta th tr uk vi zh-hans zh-hant)
+  locales: ~w(ar bg bn de en es fr el id it ja ko ms no pl pt ru sk ta th tr uk vi zh-hans zh-hant)
```

Comment:
```suggestion
  locales: ~w(ar bg bn de el en es fr id it ja ko ms no pl pt ru sk ta th tr uk vi zh-hans zh-hant)
```

Very nitpicky, but it might be nice to maintain the alphabetization of this list 🤷🏼

Reply:
👌  @kinson

Reply:
Thanks you two ❤️